webapp/src/pages/app/projects/[projectName]/index.tsx

```diff
@@ -16,6 +19,18 @@ const ProjectDetailPage = () => {
     refetchOnMountOrArgChange: true,
     skip: false,
   });
+  const [tabIndex, setTabIndex] = useState(0);
+  const tabs = useMemo(() => ['repositories', 'permissions', 'members', 'tokens', 'mirror'], []);
```

Comment:
I don't see we can take any advantage of `useMemo()` for a static array.
How about declaring it as a const at the top level?